pages/linux/shasum.md

```diff
@@ -0,0 +1,23 @@
+# shasum
+
+> Calculate or check cryptographic SHA checksums.
+
+- Calculate the SHA1 checksum for a file:
+
+`shasum {{filename}}`
+
+- Calculate the SHA256 checksum for a file:
+
+`shasum -a 256 {{filename}}`
```

Comment:
Perhaps you could use the long form of the argument name here? So instead of `-a` use `--algorithm`?